client/l2tp_client.c

```diff
@@ -1386,6 +1419,10 @@ int main(int argc, char **argv)
     // Make sure all brokers are in sane state.
     for (i = 0; i < broker_cnt; i++) {
       context_reinitialize(brokers[i].ctx);
+      if (brokers[i].broken && brokers[i].broken + 3600 < timer_now()) {
```

Comment:
Should the retry period be made configurable instead of hardcoded to 3600 seconds?

Reply:
This is not the only timeout -- there's also the 20s (I think) for the initial availability check, and the 10s for restarting the entire thing.

I considered making them all macros, but did not want the PR to get bigger than necessary. Making them CLI arguments is of course also an option, but is that really useful?